content/en/serverless/step_functions/merge-step-functions-lambda.md

```diff
@@ -111,6 +111,8 @@ For Python: Datadog Lambda Library for Python layer 99+ **or** `dd-trace-py` v2.
 
 If the layer or tracer version requirements are fulfilled, no further setup is required.
 
+<div class="alert alert-info">Ensure that some input, even if empty, is provided to the Step Functions Start Execution command for trace merging to function correctly.</div>
```

Comment:
```suggestion
<div class="alert alert-info">To ensure that trace merging functions correctly, provide input to the Step Functions Start Execution command. The input can be empty.</div>
```

Is this referring to an empty string? What does empty input mean?

Reply:
It's referring to stringified JSON input here

Reply:
Okay. Hopefully this is obvious enough.

Reply:
Ah, I see you changed my suggestion to be more specific. Excellent! Thanks.